File: TDD Boot Camp/Part 01/Code/OOPrinciples/src/com/oop/telldonotask/bad/Tester.java
package com.oop.telldonotask.bad;

public class Tester {
	public Unit convertToCentiMeters() {
		Unit actual = new Unit(10);
		double value = actual.getValue();
		double convertedValue = value/100;
		Unit converted = new Unit(convertedValue);
		return converted;
	}
	public Unit convertToMeters() {
		Unit actual = new Unit(10);
		double value = actual.getValue();
		double convertedValue = value/10000;
		Unit converted = new Unit(convertedValue);
		return converted;
	}
}

File: TDD Boot Camp/Part 01/Code/C#/OOPrinciples/TellDonotAsk/Bad/Tester.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TellDoNotAsk.Bad
{
    public class Tester
    {
        public Unit ConvertToCentiMeters()
        {
            Unit actual = new Unit(10);            
            double convertedValue = actual.Value / 100;
            Unit converted = new Unit(convertedValue);
            return converted;
        }
        public Unit ConvertToMeters()
        {
            Unit actual = new Unit(10);            
            double convertedValue = actual.Value / 10000;
            Unit converted = new Unit(convertedValue);
            return converted;
        }
    }
}
